BUILDING.md

```diff
@@ -70,10 +70,165 @@ sh ./build.sh --target=nuget && sh ./build.sh --target=samples && sh ./build.sh
 
 **Windows:**
 
-```
+```pwsh
 ./build.ps1 --target=nuget ; ./build.ps1 --target=samples ; ./build.ps1 --target=diff
 ```
 
-### Working in Visual Studio / Xamarin Studio
+## Working in Visual Studio / Xamarin Studio
```

Comment:
Could also update this since "Xamarin Studio" is gone. 😁

Reply:
nostalgia.